src/main/kotlin/domain/user/controller/StudentController.kt

```diff
@@ -1,7 +1,50 @@
 package domain.user.controller
 
-import org.springframework.web.bind.annotation.RestController
+import domain.user.dto.StudentResponseDto
+import domain.user.dto.UpdateStudentRequestDto
+import domain.user.service.UserService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
 
 @RestController
-class StudentController {
+@RequestMapping("/students/{student_id}")
+class StudentController (
+    private val userService : UserService,
+        ){
+
+    @GetMapping()
+    fun getStudent(@PathVariable student_id :Long): ResponseEntity<StudentResponseDto> {
```

Comment:
```suggestion
    fun getStudent(@PathVariable studentId :Long): ResponseEntity<StudentResponseDto> {
```
kotlin 코드의 경우 변수 명을 camelCase로 작성해주세요